draft-ietf-quic-qlog-h3-events.md

```diff
@@ -29,55 +29,22 @@ author:
     org: Protocol Labs
     email: marten@protocol.ai
     role: editor
+  - ins: L. Pardue
+    name: Lucas Pardue
+    org: Cloudflare
+    email: lucaspardue.24.7@gmail.com
+    role: editor
 
 normative:
 
   RFC9114:
     display: HTTP/3
 
   QLOG-MAIN:
-    title: "Main logging schema for qlog"
-    date: {DATE}
-    seriesinfo:
-      Internet-Draft: draft-ietf-quic-qlog-main-schema-latest
-    author:
-      -
-        ins: R. Marx
-        name: Robin Marx
-        org: KU Leuven
-        role: editor
-      -
-        ins: L. Niccolini
-        name: Luca Niccolini
-        org: Facebook
```

Comment:
given that you are at it, my affiliation will also need to change from Facebook to Meta

Reply:
not saying you have to do it. I can do it. But something that just came to mind when seeing this PR

Reply:
Make a PR 😀